pyccl/halos/massdef.py

```diff
@@ -194,6 +204,36 @@ def get_radius(self, cosmo, M, a):
             return R[0]
         return R
 
+    def translate_mass(self, cosmo, M, a, m_def_other):
```

Comment:
Could we just call `mass_translator` here?

Reply:
I couldn't be bothered to write new code so I just copied + pasted the old one. Let's just not.